### PR TITLE
[TR] PIM-5278: Fix missing translations

### PR DESCRIPTION
**Pull request by pierallard**
| Q                 | A
| ----------------- | ---
| Specs             | n
| Behats            | n
| Blue CI           | (always the same failing tests)
| Changelog updated | n
| Review and 2 GTM  | y
| Micro Demo (PO)   | y
| Migration script  | n
| Tech Doc          | n

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Form/Type/ProductCreateType.php [hunk 1]
```diff
@@ -40,7 +49,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'empty_value' => "",
                     'select2'     => true,
                     'attr'        => [
-                        'data-placeholder' => 'Choose a family'
+                        'data-placeholder' => $this->translator->trans('Choose a family')
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Form/Type/ProductCreateType.php]: no. in symfony form, all label are interpreted. if key `Choose a family` is in translations files, it's enough. 
So remove injection of translator service

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_types.yml [hunk 2]
```diff
@@ -82,6 +82,8 @@ services:
 
     pim_enrich.form.type.product_create:
         class: %pim_enrich.form.type.product_create.class%
+        arguments:
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Resources/config/form_types.yml]: to remove

**Comment by skeleton** [comment]
GTM after deletion of translator service in form type

File: src/Pim/Bundle/EnrichBundle/Form/Type/ProductCreateType.php (hunk 1 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Form/Type/ProductCreateType.php]: :+1:

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 3]
```diff
@@ -233,6 +234,7 @@ pim_enrich:
                     info:
                         no_family: No family defined. Please define a family to calculate the completeness of this product.
                         no_completeness: You just changed the family of the product. Please save it first to calculate the completeness for the new family.
+                    missing_values: "Missing values"
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: why quotes?

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.fr.yml [hunk 4]
```diff
@@ -222,6 +223,7 @@ pim_enrich:
           info:
             no_family: Aucune famille de défini. Merci de définir une famille pour calculer la complétude de ce produit.
             no_completeness: "Vous venez de modifier la famille du produit. Merci de l'enregistrer en premier pour calculer la complétude de la nouvelle famille."
+          missing_values: "Valeurs manquantes"
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.fr.yml]: Why quotes?

**Comment by grena** [comment]
Same thing than @skeleton, 2 questions and GTM.

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml (hunk 3 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: ok seen, quotes to remove

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.fr.yml (hunk 4 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.fr.yml]: ok seen, will be erased by crowdin

**Comment by skeleton** [comment]
GTM

**Pull request merged**